src/compile_torture_tests.py

```diff
@@ -58,8 +58,6 @@ def run(c, cxx, testsuite, sysroot_dir, fails, out, config='wasm'):
                              'BINARYEN_METHOD="interpret-binary"'],
   }
 
-  assert os.path.isfile(c), 'Cannot find C compiler at %s' % c
```

Comment:
Yeah. Although now that I think of it I should put `Executable()` around what we pass in here for consistency with the other tools. I'll do that and then I should be able to put these back.